src/main/scala/uk/gov/ons/bi/dataload/exports/HmrcBiCsvExtractor.scala

```diff
@@ -62,20 +34,71 @@ object HmrcBiCsvExtractor {
     log.info(s"BI index file contains ${biData.count} records.")
 
     // Extract the different sets of data we want, and write to output files
-    val legalEntities = getLegalEntities(biData)
+
+    val hmrcOutput = getHMRCOutput(biData, hmrcFile)
+    log.info(s"Writing ${hmrcOutput.count} Legal Entities to $hmrcFile")
+
+    val legalEntities = getLegalEntities(biData, legalFile)
     log.info(s"Writing ${legalEntities.count} Legal Entities to $legalFile")
-    BiCsvWriter.writeCsvOutput(legalEntities, legalFile)
 
-    val vat = getVatExploded(biData)
+    val vat = getVatExploded(biData, vatFile)
     log.info(s"Writing ${vat.count} VAT entries to $vatFile")
-    BiCsvWriter.writeCsvOutput(vat, vatFile)
 
-    val paye =  getPayeExploded(biData)
+    val paye =  getPayeExploded(biData, payeFile)
     log.info(s"Writing ${paye.count} PAYE entries to $payeFile")
-    BiCsvWriter.writeCsvOutput(paye, payeFile)
 
     // Clear cache
     biData.unpersist(false)
   }
 
+  def stringifyArr(stringArr: Column) = concat(lit("["), concat_ws(",", stringArr), lit("]"))
+
+  def getHMRCOutput(df: DataFrame, outputPath: String): DataFrame = {
+    val arrDF = df
+      .withColumn("arrVar", df("VatRefs").cast(ArrayType(StringType)))
+      .withColumn("arrPaye", df("PayeRefs").cast(ArrayType(StringType)))
+
+    val dropDF = arrDF
+      .withColumn("VatRef", stringifyArr(arrDF("arrVar")))
+      .withColumn("PayeRef", stringifyArr(arrDF("arrPaye")))
+      .drop("VatRefs","PayeRefs","arrVar", "arrPaye")
+
+    val hmrcOutput = dropDF
+      .select("id","BusinessName","TradingStyle", "PostCode",
+        "Address1", "Address2","Address3","Address4", "Address5",
+        "IndustryCode","LegalStatus","TradingStatus",
+        "Turnover","EmploymentBands","CompanyNo","VatRef","PayeRef")
+    BiCsvWriter.writeCsvOutput(hmrcOutput, outputPath)
+    hmrcOutput
+  }
+
+  def getLegalEntities(df: DataFrame, outputPath: String): DataFrame = {
```

Comment:
The naming conventions of these methods is inconsistent. You're using `getMHRCOutput` above, but you're missing the `Output` here.